--- harfbuzz-1.7.5/CMakeLists.txt	2018-02-18 22:11:35.513077800 -0600
+++ ../dl/harfbuzz-1.7.5/CMakeLists.txt	2018-01-30 13:01:18.000000000 -0600
@@ -311,7 +311,7 @@
   pkg_check_modules(PC_GLIB QUIET glib-2.0)

   find_library(GLIB_LIBRARIES NAMES glib-2.0 HINTS ${PC_GLIB_LIBDIR} ${PC_GLIB_LIBRARY_DIRS})
+  find_path(GLIBCONFIG_INCLUDE_DIR NAMES glibconfig.h HINTS ${PC_LIBDIR} ${PC_LIBRARY_DIRS} ${PC_GLIB_INCLUDEDIR} ${PC_GLIB_INCLUDE_DIRS} PATH_SUFFIXES glib-2.0/include lib/glib-2.0/include)
-  find_path(GLIBCONFIG_INCLUDE_DIR NAMES glibconfig.h HINTS ${PC_LIBDIR} ${PC_LIBRARY_DIRS} ${PC_GLIB_INCLUDEDIR} ${PC_GLIB_INCLUDE_DIRS} PATH_SUFFIXES glib-2.0/include)
   find_path(GLIB_INCLUDE_DIR NAMES glib.h HINTS ${PC_GLIB_INCLUDEDIR} ${PC_GLIB_INCLUDE_DIRS} PATH_SUFFIXES glib-2.0)

   include_directories(${GLIBCONFIG_INCLUDE_DIR} ${GLIB_INCLUDE_DIR})
@@ -785,6 +785,7 @@


 ## src/ executables
+if (HB_BUILD_UTILS)
 foreach (prog main test test-would-substitute test-size-params test-buffer-serialize hb-ot-tag)
   set (prog_name ${prog})
   if (${prog_name} STREQUAL "test")
@@ -795,6 +795,7 @@
   target_link_libraries(${prog_name} harfbuzz ${THIRD_PARTY_LIBS})
 endforeach ()
 set_target_properties(hb-ot-tag PROPERTIES COMPILE_FLAGS "-DMAIN")
+endif()

 ## Tests
 if (UNIX OR MINGW)

--- harfbuzz-1.7.5/test/api/CMakeLists.txt	2018-02-18 20:10:29.089614700 -0600
+++ ../dl/harfbuzz-1.7.5/test/api/CMakeLists.txt	2018-01-08 08:25:50.000000000 -0600
@@ -1,4 +1,4 @@
-if (HB_HAVE_GLIB)
+if (HB_HAVE_GLIB AND HB_BUILD_UTILS)
   file (READ "${CMAKE_CURRENT_SOURCE_DIR}/Makefile.am" MAKEFILEAM)
   extract_make_variable (TEST_PROGS ${MAKEFILEAM})
 
